src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -847,33 +857,24 @@ public void MSBuildEngineLogger()
             }
         }
 
-        private string _pathToMSBuildExe = Path.Combine(Directory.GetCurrentDirectory(), "MSBuild.exe");
+        //private string _pathToMSBuildExe = Path.Combine(Directory.GetCurrentDirectory(), "MSBuild.exe");
+        private string _pathToMSBuildExe = FileUtilities.CurrentExecutablePath;
 #if FEATURE_SPECIAL_FOLDERS
         private string _pathToArbitraryBogusFile = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), "notepad.exe"); // OK on 64 bit as well
 #else
         private string _pathToArbitraryBogusFile = Path.Combine(FileUtilities.GetFolderPath(FileUtilities.SpecialFolder.System), "notepad.exe"); // OK on 64 bit as well
 #endif
 
-#if FEATURE_RUN_EXE_IN_TESTS
         /// <summary>
         /// Basic case
         /// </summary>
         [Fact]
         public void GetCommandLine()
         {
-            string output = RunProcessAndGetOutput(_pathToMSBuildExe, "\"" + _pathToArbitraryBogusFile + "\"" + " /v:diag", expectSuccess: false);
-
-            Assert.True(output.Contains(_pathToMSBuildExe + " /v:diag " + _pathToArbitraryBogusFile));
-        }
+            var msbuildParameters = "\"" + _pathToArbitraryBogusFile + "\"" + " /v:diag";
+            Assert.True(File.Exists(_pathToArbitraryBogusFile));
```

Comment:
Yes it is. If that file is not there, it's better for an assert to fail in a visible way than have RunProcessAndGetOutput missbehave and churn out funky state.